scripts/retrieve-backup.sh

```diff
@@ -1,12 +1,17 @@
 #!/bin/bash
 [ $DEBUG ] && set -x
 
+if [ "${CLUSTER_ID}" == "" ];
+then
+        echo "You must specify a CLUSTER_ID for the backup process"
+        exit 1;
+fi
+
 # XXX: Set this to "echo" to for a dry-run
 DEBUG=""
 
 # Grab date / cluster name
-IFS='-' read -ra HOST <<< "${HOSTNAME:-localhost}"
-TARGET_PATH=${BACKUP_DEST:-/ndsbackup}/${HOST[0]}/$1
+TARGET_PATH=${BACKUP_DEST:-/ndsbackup}/${CLUSTER_ID}/$1
 
 echo "Retrieving backup $1 for ${HOST[0]}: ${DATE}"
```

Comment:
Should this be CLUSTER_ID instead of HOST[0]?

Reply:
I forget, but you are likely correct.

My attention to detail has been slipping lately :/